src/OmniSharp.Abstractions/Models/v1/GotoDefinition/GotoDefinitionResponse.cs

```diff
@@ -3,13 +3,14 @@
 
 namespace OmniSharp.Models.GotoDefinition
 {
-    public class GotoDefinitionResponse
+    public class GotoDefinitionResponse : ICanBeEmptyResponse
```

Comment:
it's too late to solve this one, but fundamentally I believe the mistake was returning an uninitialized object in case no match was made. we should have used better semantics to express this but it would be a breaking change now

Reply:
@filipw Hmm, could you please clarify the scenario where you think this change would be cause a break?

Reply:
We could make `IsEmpty` internal for now (along with it's interface noted above) and work with the interface.

Reply:
@dmgonch keep in mind we now publish nuget packages for Abstractions, hence we have to be careful what changes we make.

Reply:
Now that I think about it... technically speaking GotoDefinition could return mutliple results, this happens as has been adapted into LSP iirc.  This may be something we should consider making first class (returning a single definition or a list of definitions)

Reference: https://microsoft.github.io/language-server-protocol/specification#textDocument_definition

Reply:
I'm assuming you wouldn't want to bundle this change with this PR. But I made ICanBeEmptyResponse interface internal as per the other comment.